### PR TITLE
fix(report): fixed bug where export would break if report had totals

### PR DESCRIPTION
**Pull request by vjFaLk**
Please read the [Pull Request Checklist](https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist) to ensure you have everything that is needed to get your contribution merged.

### CODE REVIEW

File: frappe/desk/query_report.py [hunk 1]
```diff
@@ -243,8 +243,11 @@ def export_query():
 				# only rows which are visible in the report
 				if row and (i in visible_idx):
 					row_list = []
-					for idx in range(len(data.columns)):
-						row_list.append(row.get(columns[idx]["fieldname"],""))
+					if isinstance(row, list):
+						row_list = row
```
Comment (Alchez) [inline frappe/desk/query_report.py:247]: Don't re-assign the `row_list` object. Just append, like on line 250.

**Review by Alchez — DISMISSED** [review]

File: frappe/desk/query_report.py (hunk 1 above)
Comment (Alchez) [reply frappe/desk/query_report.py:247]: Also, maybe add a comment regarding why it would be a list.

Comment (vjFaLk) [reply frappe/desk/query_report.py:247]: Notice that the row_list is the row itself, and the "row" you get is already in the format that needs to be put in the result. i.e a list

Comment (Alchez) [reply frappe/desk/query_report.py:247]: Ah, you're right. Okay, this looks good.

**Comment by Alchez** [comment]
Also make to upstream. Another possible fix could be to turn the Total row into a dict / object.

**Comment by vjFaLk** [comment]
> Also make to upstream. Another possible fix could be to turn the Total row into a dict / object.

If you see what they're doing to the object, it's taking each key and putting it in the row. Turning the Totals row into an object would be silly.

**Comment by vjFaLk** [comment]
https://github.com/frappe/frappe/pull/6580

**Pull request merged**